Source/WebCore/Modules/reporting/ViolationReportType.h

```diff
@@ -31,6 +31,7 @@ enum class ViolationReportType : uint8_t {
     COEPInheritenceViolation, // https://html.spec.whatwg.org/multipage/origin.html#queue-a-cross-origin-embedder-policy-inheritance-violation
     CORPViolation, // https://fetch.spec.whatwg.org/#queue-a-cross-origin-embedder-policy-corp-violation-report
     ContentSecurityPolicy,
+    CSPHashReport,
     CrossOriginOpenerPolicy,
```

Comment:
These were sorted case sensitive alphabetical before, not logically. So CSP should come before Content.